inc/ADC.h

```diff
@@ -0,0 +1,49 @@
+#ifndef ADC_H_
+#define ADC_H_
+
+#define sensor_quantity 17
+#define ammount_of_recorded_values 5
```

Comment:
One more thing from me: to follow proper C coding standards, these define statements (and any other define statements) should be all caps. The reason for this, is so that when you're reading code, it's easy to distinguish between variables and constants (ie define statements). Just makes the code easier to read.